### DISCUSSION TITLE
Internal TLS

### DISCUSSION THREAD

**Discussion by oscarcarlsson** [authelia/authelia#5592] [Troubleshooting] [answered]
I have an Authelia instance running in a pod (together with a Caddy reverse proxy, internal FQDN auth.home.arpa), using TLS from my 'central' reverse proxy to my Authelia pod. My 'central' reverse proxy (also Caddy) terminates TLS for auth.example.com.

![caddy-autherlia](https://github.com/authelia/authelia/assets/359477/c9dd87cb-1857-48b1-bcf9-063c26211079)

I'm getting this error message (changed the domain names below) when trying to authenticate with Authelia:

Target URL https://auth.home.arpa:443/ is not under the protected domain example.com" method=GET path=/api/verify 

How do I correctly configure Authelia and Caddy for this? I'd like to use TLS for my internal communications if possible.

**Comment by james-d-elliott** [comment, marked as answer]
Sorry for not seeing this earlier.

You need to ensure the X-Forwarded-* headers are correct per https://www.authelia.com/integration/proxies/introduction/#required-headers
